recipes/ipyregulartable/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ipyregulartable" %}
+{% set version = "0.1.3" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ipyregulartable-{{ version }}.tar.gz
+  sha256: 85822c715131af3274fc0f5a920ac726c7c5241d923a76c8d6894255c4e4aa2a
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps
+    - cd js && yarn --production && yarn licenses generate-disclaimer > ../JS_LICENSES
+
+requirements:
+  build:
+    - yarn
+  host:
+    - jupyter-packaging
```

Comment:
Neat! Is this how we should be packaging Jupyter extensions going forward?

Reply:
It's what the cookiecutters suggest, but has hidden gotchas.... see the `yarn` dependency above. once some of the `pyproject.toml` and `pip` stuff shakes out, i hope folks will be able to use `flit`, as it has a better (as in _an_) reproducible build story... but currently doesn't support `data_files`.

Reply:
Is it worth documenting this pattern in conda-forge for people to align on it? Or is it still too much of a moving target atm?

Reply:
Nah... `grayskull` picked up the `setup_requires` just fine... the upstream issue pointed out the piece about `js` not being prebuilt in the `sdist`.